### DISCUSSION TITLE
[Feature] Is Java agent support co.elastic.clients.elasticsearch-java now?

### DISCUSSION THREAD

**Discussion by imchao9** [apache/skywalking#9896] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I want to know: is the Skywalking support  co.elastic.clients:elasticsearch-java now?
I check the [supported-list](https://skywalking.apache.org/docs/skywalking-java/v8.12.0/en/setup/service-agent/java-agent/supported-list/) and can not find this plugin：
```
    compile("co.elastic.clients:elasticsearch-java:7.17.2")
```

Is there any plan for it?

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/6490080/199429429-d835702d-c260-430c-9ef3-9e5e441b3cf9.png">
<img width="732" alt="image" src="https://user-images.githubusercontent.com/6490080/199429539-a31ef78b-7788-409a-930a-19c1a4bbad33.png">

### Use case

Skywalking support co.elastic.clients:elasticsearch-java in tracing

### Related issues

I found no

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
This is not an issue to track. If it is not on the support list, it should not be supported or tested.

**Reply by imchao9** [reply]
> This is not an issue to track. If it is not on the support list, it should not be supported or tested.

Ok, get it, so the agent not support co.elastic.clients.elasticsearch-java?
just want to confirm this, I think yes but not sure

**Reply by wu-sheng** [reply]
Again, if it is not on the doc, it should not be supported. There is no point to ask for confirmation actually. We are reading the same doc.
